torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -1009,6 +1011,92 @@ def _annotate_linear_unary(
                 _is_output_of_quantized_pattern=True,
             )
 
+    def _annotate_linear_binary(
+        self, gm: torch.fx.GraphModule, quantization_config: QuantizationConfig
+    ) -> None:
+        # linear + add
+        fused_partitions = find_sequential_partitions(
+            gm, [torch.nn.Linear, operator.add]
+        )
+        for fused_partition in fused_partitions:
+            linear_partition, binary_partition = fused_partition
+            linear_node, binary_node = self._get_output_nodes_of_partitions(
+                [linear_partition, binary_partition]
+            )
+            if len(linear_node.users) != 1:
+                # Linear Node should only has 1 user node
+                continue
+            linear_node_idx, extra_input_node_idx = self._get_input_idx_for_binary_node(
+                linear_node, binary_node
+            )
+            if (linear_node_idx is None) or (extra_input_node_idx is None):
+                continue
+            if linear_node != binary_node.args[linear_node_idx]:
+                raise ValueError(f"{linear_node} doesn't match input of binary node")
+            extra_input_node = binary_node.args[extra_input_node_idx]
+            assert isinstance(linear_node, Node)
+            if (
+                linear_node.op != "call_function"
+                or linear_node.target != torch.ops.aten.linear.default
+            ):
+                # No linear node found to be fused with add
+                continue
+            if _is_annotated([binary_node, linear_node]):
+                continue
+            self._annotate_linear_node_helper(linear_node, False, quantization_config)
+            binary_node_input_qspec_map: Dict[Node, Any] = {}
+            # We don't insert q-dq before the binary input node due to accuracy issues
+            binary_node_input_qspec_map[extra_input_node] = None
+            binary_node.meta[QUANT_ANNOTATION_KEY] = _X86InductorQuantizationAnnotation(
+                input_qspec_map=binary_node_input_qspec_map,
```

Comment:
Don't need to pass it in since `binary_node_input_qspec_map` is actual a empty dict?

Reply:
Thanks. It's fixed.